chore(macros): Remove SectionOnPage macro usage (#23059)
mdn/content: files/en-us/web/api/mediatrackconstraints/suppresslocalaudioplayback/index.md

let isLocalAudioSuppressed = displayStream.getVideoTracks()[0].getSettings().suppressLocalAudioPlayback;
```

> See [Media Streams API, Example: Constraint exerciser](/en-US/docs/Web/API/Media_Capture_and_Streams_API/Constraints#example_constraint_exerciser) for an example that allows you to play with media track constraints.

Author: Good, I think this was a leftover

> See [Media Streams API, Example: Constraint exerciser](/en-US/docs/Web/API/Media_Capture_and_Streams_API/Constraints#example_constraint_exerciser) for an example that allows you to play with media track constraints.

Author: ```suggestion
The [Constraint exerciser](/en-US/docs/Web/API/Media_Capture_and_Streams_API/Constraints#example_constraint_exerciser) example shows how to use media track constraints.
```
